tests/models/test_hf_model.py

```diff
@@ -30,6 +30,26 @@
 from tests.loggers.test_remote_uploader_downloader import DummyObjectStore
 
 
+def test_hf_tokenizer_save(tmp_path: Path, tiny_bert_model, tiny_bert_tokenizer):
+    transformers = pytest.importorskip('transformers')
+
+    trainer = get_lm_trainer(tiny_bert_model, tiny_bert_tokenizer, str(tmp_path), is_conditional_generation=True)
```

Comment:
doesn't really matter for the purposes of this test, but `is_conditional_generation` should be `False` for a bert masked language modeling model :)